### DISCUSSION TITLE
Alex313031 M105.0.5145.0 seems to crash

### DISCUSSION THREAD

**Discussion by whxuezhe** [Alex313031/Thorium-Win-AVX2#16] [Q&A]
I use Windows11 X64 home Chinese version 25151.1000 to run it, the 12th generation i9 12900H, it seems that it does not support the search of some Chinese words in Bing, or does not support opening a very small number of domestic rebuttal video sites, all of which will crash. Phenomenon; hope the positioning processing will be fixed in the next update, thank you!

**Comment by gz83** [comment]
> I use Windows11 X64 home Chinese version 25151.1000 to run it, the 12th generation i9 12900H, it seems that it does not support the search of some Chinese words in Bing, or does not support opening a very small number of domestic rebuttal video sites, all of which will crash. Phenomenon; hope the positioning processing will be fixed in the next update, thank you!

@whxuezhe What Chinese words did you search for in bing? What will the browser do when it can't search? And what is the video website you are talking about, can you provide the address for testing?

**Reply by whxuezhe** [reply]
@gz83 Tianyi cloud disk and Sogou search, the key words to refute the video website are: Nunu Cinema and the Alliance of Whore Prostitutes (1. https://www.nunuyy1.org/2. https://www.bpzhe.com/), one The probability of crashing by double-clicking the Thorium icon is very low.

**Reply by gz83** [reply]
> @gz83 Tianyi cloud disk and Sogou search, the key words to refute the video website are: Nunu Cinema and the Alliance of Whore Prostitutes (1. https://www.nunuyy1.org/2. https://www.bpzhe.com/), one The probability of crashing by double-clicking the Thorium icon is very low.

@whxuezhe I used the keyword you gave here to search in Chinese in bing, and it can be searched normally. In addition, regarding the video website, my test here is also normal, and it can be opened normally.

By the way, is it convenient for you to test with Chromium to see if these problems exist?

**Reply by whxuezhe** [reply]
@gz83 I tried 105.0.5116.0&5148.0 with all codec and widevine based on Chromium Author (chromium) and everything is normal, except our thorium (the browser built by Thorium does not work)

**Reply by gz83** [reply]
> @gz83 I tried 105.0.5116.0&5148.0 with all codec and widevine based on Chromium Author (chromium) and everything is normal, except our thorium (the browser built by Thorium does not work)

@whxuezhe Please use vanilla Chromium for testing

**Reply by whxuezhe** [reply]
@gz83 It should be speculated that the 12-generation CPU is heterogeneous in size and core, resulting in inadequate instruction set compatibility and adaptation; This problem also exists in the Opera browser Developer update channel version.

**Comment by whxuezhe** [comment]
@gz83 5153 Temporarily solved my previous problem, at least opening Nunu Cinema is normal, but the initial loading speed of the episode list and video is not as fast as Microsoft's magic-modified chromium-core Edge browser. (Please pay attention to the previous problem when you continue to test, because this problem is one-sidedly considered to be a compatibility unknown problem and when the execution handle is executed, the code pointer encounters the function declaration API function crashes)

**Reply by gz83** [reply]
@whxuezhe I'll discuss this with Alex to see if there is a solution, I can't reproduce the issue you're having on my side.
@Alex313031

**Comment by Alex313031** [comment]
@gz83 Since we changed our api keys and I recently added a patch for widevine, this shouldn't be happening. We will test on next release.

**Reply by whxuezhe** [reply]
@Alex313031 @gz83 Other reminders about building, compiling and releasing; crash and flashback: When Vulkan rendering is turned on, there may be crashes caused by assertions, such as debugging code participating in red screen assertions.

**Reply by gz83** [reply]
> @Alex313031 @gz83 Other reminders about building, compiling and releasing; crash and flashback: When Vulkan rendering is turned on, there may be crashes caused by assertions, such as debugging code participating in red screen assertions.

@whxuezhe Have you used the AVX version?  Does the AVX version have these issues?
